server/app/assets/stylesheets/uswds/_uswds-theme-custom-styles.scss

```diff
@@ -32,3 +32,20 @@ See the radio-option below as an example.
 #program-image-upload-file-input .usa-file-input {
   max-width: none;
 }
+
+// Placeholder styles for cards.
+/* stylelint-disable */
+@media all and (min-width: 55em) {
+  .tablet-lg\:grid-col-6 {
+    flex: 0 1 auto !important;
+    width: 50% !important;
+  }
+}
+
+@media all and (min-width: 87.5em) {
+  .widescreen\:grid-col-4 {
+    flex: 0 1 auto !important;
+    width: 33.3333333333% !important;
+  }
+}
```

Comment:
I'd advise skipping these styles for now for a couple reasons:
1) there are a lot of arbitrary-ish constants in here that could interact in unexpected ways with other classes.
2) !important should generally be avoided because it breaks the natural cascading of styles and can make it hard to reason about the applied styles
3) rather than overriding the USWDS styles, we should add additional classes on top of the USWDS classes. By changing USWDS styles, we could be causing unintended consequences for other componenets that use these classes

If that means the cards look a little wonky while the final styles are still pending, I think that's fine

Reply:
Done